pommerman/network/ion_server/network.py

```diff
@@ -0,0 +1,205 @@
+#!/usr/bin/env python
+"""IonServer Network handler - This contains functions responsible Server-Client communication (You shouldn't use this file directly)"""
+
+# Imports
+from asyncio import new_event_loop, set_event_loop, get_event_loop
+from websockets import serve
+from threading import Thread
+from .constants import SPCommands as spc
+from .constants import NetworkCommands as nic
+from time import time, sleep
+from os import getcwd, path
+from re import fullmatch
+from gzip import compress
+from rapidjson import loads, dumps, load
+from json import JSONDecodeError
+from uuid import uuid4
+
+# Global variables
+CONCURRENTLY_LOOKING = {"room": {}, "noroom": []}
+PLAYER_WS = {}
+MATCH_PROCESS = {}
+THREAD_LOCK = False
```

Comment:
The only one of these that's clear from the name is THREAD_LOCK. Please either change the names of the others (preferred) or provide a comment saying what they are.

Reply:
Done